test/e2e/helpers.js

```diff
@@ -981,6 +981,26 @@ async function openMenuSafe(driver) {
   }
 }
 
+async function enableExperimentalMode(driver) {
```

Comment:
we should use page object modal for settings, see example here : https://github.com/MetaMask/metamask-extension/blob/37e79458f3fb8299be2c5d583796fd79f28cb60b/test/e2e/tests/account/snap-account-settings.spec.ts#L30

Reply:
removed